### ISSUE TITLE
Feature Request: HOOBS configuration

### ISSUE THREAD

**Issue by Drewbacca2** [bloomkd46/homebridge-iRobot#93]
### Is your feature request related to a problem? Please describe:

When using HOOBS instead of Homebridge, the configuration options are not available. I believe this is because the plugin does not properly use the config schema, but I got this from another troubleshooting website. It can be done, but it has to be done manually in the advanced section. Unfortunately, there's no example of all the options available in the plug-in or how to use them. I have a copy of my config prior to the 3.0 beta releases, but I'm not sure what changes came with the beta. Is there a way to see what the config page looks like in Homebridge without installing Homebridge?

### Describe the solution you'd like:

Include support for config schema so that HOOBS customization is possible.

### Describe alternatives you've considered:

Provide sample template of advanced setup page on Github site that can be copied and filled in.

### Node Version (optional):

16.14.0

### NPM Version (optional):

8.3.1

### Homebridge Version (optional):

HOOBS 4.1.22

### Plugin Version:

3.0.0-beta.2

### Operating System (optional):

MacOS

**Comment by bloomkd46** [comment]
The biggest change is there is an array named sensors that lets you add binfull or running sensors. I fixed the plug-in so that it will run without them

**Comment by Drewbacca2** [comment]
Yeah, I saw that in the Beta 2 notes. I installed Beta 2 and could not get my 980 to connect, no matter how hard I tried. My J7 connected great though, although I'm not able to get anything to work using the Automatic method. I have to add in the manual configuration for them. I'm no longer running Homebridge and am using a Raspberry Pi 4b with HOOBS installed to try to simplify things. Unfortunately, your plugin, and a couple of the others I use, doesn't have all of the customization setup show up automatically on the screen, so I have to use the advanced area to manually type everything in. If you could figure out how to fix that, or provide some kind of template that's current with the updates, then it would be a little easier to make sure I'm not missing anything.

At this point, I can either use your current release, and my 980 just disconnects after running, meaning I have to restart the plugin every evening so it will run the next day, or I can just run the J7. I'm actually going to temporarily use the Roomba2 plugin, which doesn't have some of the features yours does. I have to run two different versions of that because I have to change the default cipher to work with the J7, which disables the 980. So I run one plugin with the original (AES_128-SHA256), and a second with the corrected one (TLS_AES_256_GCM_SHA384).
